examples/action_dann_lightn/model.py

```diff
@@ -51,7 +55,7 @@ def get_config(cfg):
     return config_params
 
 
-def get_feat_extractor(model_name, num_classes, num_channels):
+def get_feat_extractor(model_name, image_modality, attention, num_classes):
```

Comment:
Why don't put this into the kale core API?
A key consideration is whether it is reusable/generic or only specific to **this example**. If it is reusable, it should be put in the core API. It seems to be usable, e.g. for other videos.
We can discuss if you are unsure.